packages/client/package.json

```diff
@@ -37,6 +37,8 @@
     "@osdk/client.unstable": "workspace:*",
     "@osdk/client.unstable.osw": "workspace:*",
     "@osdk/generator-converters": "workspace:~",
+    "@osdk/internal.foundry.core": "workspace:~",
```

Comment:
So this is an example of us moving things out of dev dependencies so they don't get bundled in right? Or did you do this for another reason?

Reply:
When client was on tsup, rollup was embedding the bits we used into the client's .js file. Since we are not using rollup on the client any more, if we didn't do this, people downstream would have errors trying to run our code.